tasks/skylab/LoomUtils.wdl

```diff
@@ -196,3 +196,100 @@ task AggregateSmartSeq2Loom {
       maxRetries: 1
     }
 }
+
+
+task SingleNucleiSmartSeq2LoomOutput {
+  input {
+    #runtime values
+    String docker = "quay.io/humancellatlas/secondary-analysis-loom-output:0.0.7"
+    # the gene count file "<input_id>_rsem.genes.results" in the task results folder call-RSEMExpression
+    # file named "<input_id>_QCs.csv" in the folder  "call-GroupQCOutputs/glob-*" of the the SS2  output
+    Array[File] smartseq_qc_files
+    # introns counts
+    File introns_counts
+    # exons counts
+    File exons_counts
+    # annotation file 
+    File annotation_introns_added_gtf
+    # name of the sample
+    String input_id
+    String? input_name
+    String? input_id_metadata_field
+    String? input_name_metadata_field
+
+    String pipeline_version
+    Int preemptible = 3
+    Int disk = 200
+    Int machine_mem_mb = 18
+    Int cpu = 4
+  }
+
+  meta {
+    description: "This  task will converts some of the outputs of Smart Seq 2 pipeline into a loom file"
```

Comment:
```suggestion
    description: "This task will convert some of the outputs of the MultiSampleSmartSeq2SingleNuclei pipeline into a loom file"
```

Or change to correct workflow name, I'm not sure if it's used in mutlisample or single sample. Also, maybe we could clarify what we mean by "Some of the outputs" if it can be summarized briefly enough.

Reply:
Changed.